dev_tools/rubocop/rails-base.yml

```diff
@@ -0,0 +1,2 @@
+require:
```

Comment:
Other files are referencing `rails.yml`. Does this file need to be renamed?

Reply:
Whoops - renamed it and forgot to push it through 😅